specification/newrelic/NewRelic.Management/examples/Organizations_List_MaximumSet_Gen.json

```diff
@@ -19,8 +19,7 @@
               "billingSource": "AZURE"
             }
           }
-        ],
-        "nextLink": "http://testlink.com"
```

Comment:
>"http://testlink.com" [](http://example.com/codeflow?start=20&length=21)

Instead of removing should not it be null here?

Reply:
Done